tendrl/commons/objects/cluster/atoms/setup_cluster_alias/__init__.py

```diff
@@ -0,0 +1,74 @@
+import time
+import uuid
+
+from tendrl.commons import objects
+from tendrl.commons.objects.job import Job
+from tendrl.commons.utils import log_utils as logger
+
+
+class SetupClusterAlias(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(SetupClusterAlias, self).__init__(*args, **kwargs)
+
+    def run(self):
+        integration_id = self.parameters['TendrlContext.integration_id']
+        if "provisioner/%s" % integration_id not in NS.node_context.tags:
+            return True
+
+        short_name = self.parameters.get('Cluster.short_name', None)
+        if short_name in [None, ""]:
+            logger.log(
+                "debug",
+                NS.publisher_id,
+                {
+                    "message": "No need to create alias"
+                    "as short_name not provided for cluster"
+                },
+                job_id=self.parameters['job_id'],
+                flow_id=self.parameters['flow_id']
+            )
+            return True
+
+        _job_id = str(uuid.uuid4())
+        payload = {
+            "tags": ["tendrl/integration/monitoring"],
+            "run": "monitoring.flows.SetupClusterAlias",
+            "status": "new",
+            "parameters": self.parameters,
+            "type": "monitoring"
+        }
+        Job(
+            job_id=_job_id,
+            status="new",
+            payload=payload
+        ).save()
+
+        # Wait for 2 mins for the job to complete
+        loop_count = 0
+        wait_count = 24
+        while True:
+            if loop_count >= wait_count:
+                logger.log(
+                    "info",
```

Comment:
Why the log level is info? its a failure scenario right?

Reply:
This is a job update and should be info type